### DISCUSSION TITLE
[v7.0.010 build 14684] snipe-it – Docker subdirectory

### DISCUSSION THREAD

**Discussion by Bitskay** [snipe/snipe-it#17556] [Q&A]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Hi there

trying to install snipe-it with docker run on a linux machine. It basically works. But I have a few things thats I am hanging on. 
I’d like to use subdirectories which is descibed here: https://snipe.it.readme.io/docs/subdirectories.
I made the changes and it works as well. But now I have the problem – it’s not persistence.
I could make volumes over and over, but it’s not my goal and what happens if there are updates for these directories?
I do have volumes beside standard volume var/lib/snipeit: /etc/apache2 (because for changes for subdirectories) and /var/www/html/public (because for changes for subdirectories in .htaccess).
Is there a way not to have volumes for /apache2 and /public? Or at least no /public. I can’t make a volume only with .htaccess, because its a file no directory.

Hope someone can help me.

### Reproduction steps

Can't make changes for subdirectory to persistent without making volumes which could make troubles after geeting updates apache or inside directory /public.

### Expected behavior

Get a solution to make these files persistent without having full directory /public and /apache2 on two volumes.

### Snipe-IT Version

v7.0.010 build 14684

### Operating System

Ubuntu

### Web Server

apache

### PHP Version

-

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
